satellite/satellitedb/database.go

```diff
@@ -71,6 +72,11 @@ func (db *DB) BandwidthAgreement() bwagreement.DB {
 // 	return &accountingDB{db: db.db}
 // }
 
+// Irreparabledb is a getter for DB repository
+func (db *DB) Irreparabledb() irreparabledb.DB {
```

Comment:
The shorthands are capitalized, so `IrreparableDB` (https://github.com/golang/go/wiki/CodeReviewComments#initialisms), however, in this case `db` is already implict, so you can use `Irreparable()`.

Reply:
done